peachjam/views/documents.py

```diff
@@ -250,6 +253,36 @@ def get_context_data(self, **kwargs):
         return context
 
 
+@method_decorator(add_slash_to_frbr_uri(), name="setup")
+class SimilarDocumentsView(DetailView):
+    template_name = "peachjam/_similar_documents.html"
+    slug_url_kwarg = "frbr_uri"
+    slug_field = "expression_frbr_uri"
+    model = CoreDocument
+
+    def get_similar_documents(self):
+        embedding = get_object_or_404(DocumentEmbedding, document_id=self.object.pk)
+        similar_docs = (
+            DocumentEmbedding.objects.exclude(document_id=self.object.pk)
```

Comment:
exclude this document's work, as well, to avoid getting historical versions of the current document.

Reply:
done